x-pack/legacy/plugins/lens/public/xy_visualization_plugin/xy_config_panel.tsx

```diff
@@ -143,6 +142,30 @@ export function XYConfigPanel(props: VisualizationProps<State>) {
         </EuiFormRow>
       )}
 
+      <EuiFormRow
+        label={i18n.translate('xpack.lens.xyChart.splitSeries', {
+          defaultMessage: 'Split series',
+        })}
+      >
+        <MultiColumnEditor
+          accessors={state.splitSeriesAccessors}
+          datasource={datasource}
+          dragDropContext={props.dragDropContext}
+          onAdd={accessor =>
+            setState({ ...state, splitSeriesAccessors: [...state.splitSeriesAccessors, accessor] })
+          }
+          onRemove={accessor =>
+            setState({
+              ...state,
+              splitSeriesAccessors: state.splitSeriesAccessors.filter(col => col !== accessor),
+            })
```

Comment:
Based on our discussion earlier, it seems like this is the best place to write the conditional logic for adding a new column if the last column was removed